assets/panel/debugger.properties

```diff
@@ -537,17 +545,29 @@ sourceSearch.search=Search sources…
 # message when the query did not match any of the sources.
 sourceSearch.noResults2=No results found
 
-# LOCALIZATION NOTE (ignoreExceptions): The pause on exceptions button tooltip
+# LOCALIZATION NOTE (ignoreExceptionsTooltip): The pause on exceptions button tooltip
+# when the debugger will not pause on exceptions.
+ignoreExceptionsTooltip=Ignore exceptions. Click to pause on uncaught exceptions
+
+# LOCALIZATION NOTE (ignoreExceptionsItem): The pause on exceptions button description
 # when the debugger will not pause on exceptions.
-ignoreExceptions=Ignore exceptions. Click to pause on uncaught exceptions
+ignoreExceptionsItem=Ignore exceptions
```

Comment:
The question about case still stands: is this supposed to use title case (like `pauseButtonItem`) or sentence case? I don't have visibility on the feature, so it might be indeed expected.

Reply:
I think sentence case is fine. It's what we're using for other lists:


![Screen Shot 2017-11-07 at 9.06.36 AM.png](https://shipusercontent.com/8c7c0826cb2075272807fc40b8a14b97/Screen%20Shot%202017-11-07%20at%209.06.36%20AM.png)
![Screen Shot 2017-11-07 at 9.06.30 AM.png](https://shipusercontent.com/22d1f7b59c0ef7264a207a48c5f6e212/Screen%20Shot%202017-11-07%20at%209.06.30%20AM.png)

Reply:
So, pauseButtonItem needs fixing instead ("Pause on Next Statement", title case)?